practical-07/src/store/actions.js

```diff
@@ -0,0 +1,98 @@
+import axios from "axios";
+import route from "../route/route";
+
+export default {
+    fetchData({ commit }) {
+        axios
+            .get("https://testapi.io/api/dartya/resource/cardata")
+            .then((res) => {
+                commit("GET_CAR_DATA", res.data.data);
+            })
+            .catch((error) => {
+                alert(error);
+                console.log(error);
+            });
+    },
+
+    deleteCarDetails({ dispatch }, id) {
+        axios
+            .delete(`https://testapi.io/api/dartya/resource/cardata/${id}`)
+            .then((response) => {
+                if (response.status === 204) {
+                    dispatch("fetchData");
+                }
+            })
+            .catch(() => {
+                alert("can't delete at this moment");
+            });
+    },
+
+    addData({ dispatch }, data) {
+        axios
+            .post("https://testapi.io/api/dartya/resource/cardata", data)
```

Comment:
Codacy has a fix for the issue: [Delete `······`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
      .post("https://testapi.io/api/dartya/resource/cardata", data)
```